client/src/components/TaskList.tsx

```diff
@@ -54,6 +59,14 @@ export const TaskList: React.FC<any> = ({ tasks }) => {
           })
         }
       </IonList>
+      <IonToast
```

Comment:
@wtrocki these changes now show an error for deleting tasks and an error also gets shown for updating tasks when the user is not authorized.

Reply:
Yes. Sorry for ignorance - I forgot that we have errors sent already sent from server
Dialog is an actual bonus. Good work